package/Dockerfile.hub-mcscontroller

```diff
@@ -0,0 +1,7 @@
+FROM scratch
+
+WORKDIR /var/submariner
+
+COPY bin/hub-mcscontroller /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/hub-mcscontroller", "-ns", "submariner-broker"]
```

Comment:
The default namespace is `submariner-k8s-broker`

Reply:
thanks for catching it - my memory's not what it used to be ;-)

Reply:
@donaldh Fixed. Please merge if acceptable